protobuf/module_msg.proto

```diff
@@ -972,3 +972,17 @@ message VXLANEncapArg {
 message WildcardMatchArg {
   repeated WildcardMatchField fields = 1; /// A list of WildcardMatch fields.
 }
+
+/**
+ * The ARP Responder module is responding to ARP requests and can optionally learn new MAC's-IP's mapping
```

Comment:
Done, yes it can be used for a proxy ARP and it can also be used later for "full" ARP layer implementation to the local host.